### DISCUSSION TITLE
402: PAYMENT_REQUIRED

### DISCUSSION THREAD

**Discussion by tangjinzhou** [vercel/community#4509] [Help] [answered]
### Summary

[402: PAYMENT_REQUIRED](https://hook-form-site.vercel.app/)

But i have enough resource

![image](https://github.com/vercel/community/assets/6937879/be80f956-bcdd-4037-a7a2-d03c8b88dd11)

### Steps to Reproduce

visit https://hook-form-site.vercel.app/

**Comment by amyegan** [comment, marked as answer]
It looks like the deployment belongs to a Pro team that was [paused](https://vercel.com/guides/why-is-my-account-deployment-blocked). You can add a valid payment method to reactivate the account. 

If the project is eligible to deploy as a personal or non-commercial project on a Hobby account, then you can [transfer the project](https://vercel.com/docs/projects/overview#transferring-a-project) to your personal Hobby account. For more info, please review the [Fair Use Guidelines](https://vercel.com/docs/limits/fair-use-guidelines#) and [Terms of Service](https://vercel.com/legal/terms#fair-use).
